.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "🐛 Bug Report"
+about: Report a bug if something isn't working as expected in Sitewide Sales.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed or left unattended.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Screenshots**
+If applicable, please attach a screenshot to make your issue clearer.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Isolating the problem (mark completed items with an [x]):**
+- [ ] I have deactivated other plugins and confirmed this bug occurs when only the Sitewide Sales plugin is active.
```

Comment:
Not sure if we want to say "with only the Sitewide Sales plugin and relevant module plugins active" if an issue is related to a particular integration

Reply:
I just made this improvement. Thank you!